src/apps/lwaftr/lwaftr.lua

```diff
@@ -192,7 +192,7 @@ function LwAftr:new(conf)
    o.hairpinning = conf.hairpinning
    o.icmpv6_rate_limiter_n_packets = conf.icmpv6_rate_limiter_n_packets
    o.icmpv6_rate_limiter_n_seconds = conf.icmpv6_rate_limiter_n_seconds
-   o.inet_mac = conf.inet_mac
+   o.inet_mac = conf.inet_mac or ethernet:pton("00:00:00:00:00:00")
```

Comment:
Not in the context of the app network, because it will be unconditionally overwritten by the ARP app, so the important thing is to have 6 bytes that can be copied into an ethernet header without crashing; they'll never hit the outside world. If you'd prefer to see a different default despite that, please say what it is. :-)